app/src/main/res/values-fa/strings.xml

```diff
@@ -0,0 +1,173 @@
+
+    <!-- Labels -->
+    <string name="apps">کاره‌ها</string>
+    <string name="sessions">جلسات</string>
+    <string name="edit_session">ویرایش جلسه</string>
+    <string name="filesystems">فایل‌سیستم‌ها</string>
+    <string name="edit_filesystem">ویرایش فایل‌سیستم</string>
+    <string name="settings">تنظیمات</string>
+    <string name="help">راهنما</string>
+    <string name="toc">شرایط و ضوابط</string>
+
+    <!-- Buttons -->
+    <string name="button_continue">ادامه</string>
+
+    <!-- Menus -->
+    <string name="app_info">اطلاعات کاره</string>
+    <string name="stop_app">توقف کاره</string>
+    <string name="kill_service">Kill Service</string>
+    <string name="edit">ویرایش</string>
+    <string name="delete">پاک‌کردن</string>
+    <string name="save">ذخیره‌کردن</string>
+    <string name="add">افزودن</string>
+    <string name="refresh">تازه‌سازی</string>
+
+    <!-- Hints -->
+    <string name="hint_filesystem_name">نام فایل‌سیستم</string>
+    <string name="hint_session_name">نام جلسه</string>
+    <string name="hint_username">نام کاربری</string>
+    <string name="hint_password">گذرواژه</string>
+    <string name="hint_vnc_password">گذرواژه VNC</string>
+    
+    <!-- Errors -->
+    <string name="error_session_name">باید نام جلسه را وارد کنید</string>
+    <string name="error_filesystem_name">باید نام فایل‌سیستم را وارد کنید</string>
+    <string name="error_username">باید نام کاربری را وارد کنید</string>
+    <string name="error_empty_field">همه فیلدها باید پر شوند!</string>
+    <string name="error_app_description_not_found">توضیحاتی برای این کاره پیدا نشد. تازه‌سازی کنید.</string>
+    <string name="error_vnc_password_length_incorrect">گذرواژه VNC باید بین ۶ تا ۸ نویسه باشد!</string>
+    <string name="error_username_invalid">نام کاربری حاوی نویسه‌های غیرمجاز است</string>
+    <string name="error_password_invalid">گذرواژه حاوی نویسه‌های غیرمجاز است</string>
+    <string name="error_vnc_password_invalid">گذرواژه VNC حاوی نویسه‌های غیرمجاز است</string>
+
+    <!-- Prompts -->
+    <string name="prompt_filesystem">فایل‌سیستم:</string>
+    <string name="prompt_service_type">نوع خدمت:</string>
+    <string name="prompt_client_type">نوع کارخواه:</string>
+    <string name="prompt_app_connection_type_preference">لطفا نوع اتصال را تعیین کنید: </string>
+
+    <!-- Notification strings -->
+    <string name="services_notification_channel_id">خدمات یوزرلند</string>
+    <string name="services_notification_channel_name">خدمات</string>
+    <string name="services_notification_channel_description">خدمات مداوم</string>
+    <string name="service_notification_title">یوزرلند</string>
+    <string name="service_notification_description">یوزرلند در حال اجرای خدمتی در پس‌زمینه است.</string>
+
+    <!-- Progress bar messages -->
+    <string name="progress_bar_start_step">در حال آماده‌سازی برای شما…</string>
+    <string name="progress_basic_app_setup">در حال تنظیم اطلاعات پایگاه داده کاره&#8230;</string>
```

Comment:
`8230#&;` needs to remain together. You have separated it into `8230#&` and `;`. The `;` is an xml special character delimiter. There are several other strings that need this fix as well.

Reply:
There is no problem. You just see it this way due to mix of RTL and LTR text. If you select the text, you will see there is no `8230#&;` is all together.

Reply:
Yep, you're right. My bad!